jekyll/_cci2/server-3-operator-orbs.adoc

```diff
@@ -41,17 +41,27 @@ circleci orb list --host <your server install domain> --token <your api token>
 ## Import a public orb
 To import a public orb to your local server orb repository:
 
-[source,bash]
-----
-circleci admin import-orb ns[/orb[@version]] --host <your server installation domain> --token <your api token>
-----
+```bash
+circleci admin import-orb ns[orb[@version]] --host <your server installation domain> --token <your api token>
+```
 
 ## Fetch a public orb’s updates
 To update a public orb in your local server orb repository with a new version, run:
 
 [source,bash]
 ----
-circleci admin import-orb ns[/orb[@version]] --host <your server installation domain> --token <your api token>
+circleci admin import-orb ns[orb[@version]] --host <your server installation domain> --token <your api token>
 ----
 
-For more orb information, please refer to the https://circleci.com/docs/2.0/orb-intro/#quick-start[Orb docs] for the cloud product.
+## Using orbs behind a proxy
+
+When importing orbs, the CLI must be able to talk to the server installation and to circleci.com. If you want to do this when using a server installation behind a proxy, the CLI needs to be configured to use the proxy to make those requests to `circleci.com` and to not proxy requests to the server install. Here is an example of how this could look:
+
+```bash
+export NO_PROXY=server.example.com
+export HTTPS_PROXY=http://proxy.example.com:3128
+export HTTP_PROXY=http://proxy.example.com:3128
+circleci admin import-orb ns[/orb\[@version]] --host <your server installation domain> --token <your api token>
```

Comment:
```suggestion
circleci admin import-orb ns[orb\[@version]] --host <your server installation domain> --token <your api token>
```

May wish to strip out the slash to make it consistent with the other commands. Or to add the slash to the others?